Cargo.toml

```diff
@@ -47,7 +47,7 @@ atomic-waker = "1.1.2"
 axum = "0.7.9"
 axum-extra = "0.9.6"
 base64 = "0.22.1"
-bincode = "2.0.0-rc.3"
+bincode = "=2.0.0-rc.3"
```

Comment:
Please make a new ticket to upgrade `bincode` to 2.0 GA, and `// TODO(RSDK-...): ...` it here. That ticket can depend on https://viam.atlassian.net/browse/RSDK-9716 which will move us to Rust 1.85+ at which point we won't need the pin anymore, so it would be nice to let it advance naturally from then on.

Reply:
Yes I don't know what this crate is doing in our tree. It seems to be reference by ota for native build but I don't think we support it? @mattjperez  can confirm whether the native code for OTA update the binary